src/main/java/seedu/edulink/logic/commands/EditCommand.java

```diff
@@ -116,10 +114,10 @@ private static Student createEditedPerson(Student studentToEdit, EditPersonDescr
         Id updatedId = editPersonDescriptor.getId().orElse(studentToEdit.getId());
         Major updatedMajor = editPersonDescriptor.getMajor().orElse(studentToEdit.getMajor());
         Intake updatedIntake = editPersonDescriptor.getIntake().orElse(studentToEdit.getIntake());
-        Grade updatedGrade = editPersonDescriptor.getGrade().orElse(studentToEdit.getGrade());
+        Set<Grade> updatedGrades = editPersonDescriptor.getGrades().orElse(studentToEdit.getGrades());
```

Comment:
I believe there is no need for the orElse. Just return editPersonDescriptor.getGrades()